src/ontology/oba-edit.obo

```diff
@@ -93,12 +72,12 @@ id: OBA:0000036 ! body fluid levels
 is_a: OBA:0000001 ! biological attribute
 
 [Term]
-id: OBA:0000037 ! brood size
+id: OBA:0000037
 name: obsolete brood size
 is_a: OBA:0000001 ! biological attribute
 
 [Term]
-id: OBA:0000039 ! buoyancy
+id: OBA:0000039
 name: obsolete buoyancy
 is_a: OBA:0000001 ! biological attribute
```

Comment:
If this is obsolete, don't we need to remove the classification?

Reply:
Oops some qc seems to be not running! FIxing that now.